austin-solution.js

```diff
@@ -0,0 +1,14 @@
+function sum(){
```

Comment:
sum what? sum body once told me the world was gunna roll me. 
more specific function signatures are better 💯